js/main.js

```diff
@@ -826,14 +826,22 @@ function createAttributeBalanceEntry(balanceElement, getEffectFn, getEffectsFn,
 
     const balanceEntryElement = Dom.new.fromTemplate('balanceEntryTemplate');
     const domGetter = Dom.get(balanceEntryElement);
-    domGetter.byClass('name').textContent = '(' + name + ')';
+    const descriptionElement = domGetter.byClass('name');
+    const updateDescription = () => {
+        const description = typeof name === 'function' ? name() : name;
+        descriptionElement.textContent = '(' + description + ')';
+    };
+
+    updateDescription();
+
     domGetter.byClass('operator').textContent = effectType.operator;
     attributeBalanceEntries.push({
         element: balanceEntryElement,
         effectType: effectType,
         getEffect: getEffectFn,
         getEffects: getEffectsFn,
         isActive: isActiveFn,
+        updateDescription,
```

Comment:
```suggestion
        updateDescription: updateDescription,
```
Bitte keine [Shorthand Properties](https://ui.dev/shorthand-properties) verwenden, das führt zu kruden Koppelungen von lokalen Variablennamen in herumgereichten Objekten.